srcpkgs/python-pyfiglet/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'python-pyfiglet'
+pkgname=python-pyfiglet
+version=0.8.0
+revision=1
+archs=noarch
+wrksrc="pyfiglet-${version}"
+build_style="python-module"
+pycompile_module="pyfiglet"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python"
+short_desc="Implementation of figlet written in Python "
+maintainer="Illia Shestakov <ishestakov@airmail.cc>"
+license="MIT"
+homepage="https://github.com/pwaller/pyfiglet"
+distfiles="https://github.com/pwaller/pyfiglet/archive/v${version}.tar.gz"
+checksum=0331d8cd949b0def033fce08c9b1f62006b8d1e1dce578dd28fdb02ae0f827f1
+
+post_install() {
+	vlicense LICENSE
+	vdoc README
+}
+
+python3-pyfiglet_package() {
+	archs=noarch
+	pycompile_module="pyfiglet"
+	depends="python3"
+	pkg_install() {
+		vmove usr/lib/python3*
```

Comment:
Please fix indentation and put `usr/lib/python3*` in double quotes

Reply:
Done